.github/fedora/contour.spec

```
@@ -27,6 +27,7 @@ BuildRequires:  ninja-build
 BuildRequires:  pkgconf
 BuildRequires:  qt5-qtbase-devel
 BuildRequires:  qt5-qtbase-gui
+BuildRequires:  qt5-qtmultimedia-devel
```

```suggestion
BuildRequires:  qt5-qtmultimedia-devel

Requires: qt5-qtmultimedia
```

Don't we also need to add this to the runtime dependencies? (probably alphabetically sorted in the Requires:-list below).